pkg/ui/chooser/embed/index.html

```diff
@@ -65,10 +78,16 @@
         </div>
 
         <ul id="namespaces-list">
-            {{ range .Namespaces }}
-            <li>
-                <a href="#" onclick="window.location = window.location.origin + {{if eq .Code "default"}}'/'{{else}}'/ns/{{.Code}}/'{{end}}">{{.Description}}</a>    
+            <li class="selected-namespace">
+                <i id="current-icon" class="Icon__container icon-long-arrow-right"></i>
+                <b id="current-namespace">{{.CurrentNamespace.Description}}</b>
             </li>
```

Comment:
It seems like this could cause the current namespace to "move" after clicking (from middle of list to top) -- maybe this logic should be inside the loop instead, so it stays in the same place?